diff --git a/source/scripts/phantom.sh b/source/scripts/phantom.sh
index f027a81..d77823a 100644
--- a/scripts/phantom.sh
+++ b/scripts/phantom.sh
@@ -1,7 +1,7 @@
 #!/bin/bash
 #XXX change this to your needs
 USER=phantom_user
-TUNCTL=/path/to/tunnel_binary
+TUNCTL=./tunnel
 
 start() {
 	echo start
diff --git a/source/src/test/gencerts.sh b/source/src/test/gencerts.sh
index e00e8ec..7ebecdb 100755
--- a/src/test/gencerts.sh
+++ b/src/test/gencerts.sh
@@ -1,4 +1,14 @@
-HOSTS="faui00a faui00b faui00c faui00d faui00e faui00f faui00g faui00h faui00i faui00j faui00k faui03a faui00l faui00m faui00n faui00o faui00p faui00q faui00r faui00s faui00t"
+#!/bin/bash
+
+if [ -n "${1}" ]
+then
+	HOSTS="$1"
+else
+	echo "Enter the hosts for which you wish to generate a config file for sperated by a space:"
+	read HOSTS
+fi
+
+echo $HOSTS
 
 CERTPOSTS="cc pbc rc"
 for i in $HOSTS;
@@ -8,7 +18,14 @@ for i in $HOSTS;
 		openssl genrsa 1024 > "$i-privkey-$j.pem";
 		yes XX | openssl req -new -x509 -nodes -sha1 -days 365 -key "$i-privkey-$j.pem" > "$i-$j.pem";
 	done
-IP=`host -t A $i | cut -d ' ' -f 4`
+
+	# seccloud specific!
+	if ! host -t A "$i"
+	then
+		IP=`echo "$i" | cut -b 4- | sed -e 's/-/./g'`
+	else
+		IP=`host -t A $i | cut -d ' ' -f 4`
+	fi
 cat << EOF > $i.conf
 <?xml version="1.0" encoding="iso-8859-1" ?>
 
